metricbeat/mb/mb.go

```diff
@@ -106,6 +107,7 @@ type MetricSet interface {
 	HostData() HostData                  // HostData returns the parsed host data.
 	Registration() MetricSetRegistration // Params used in registration.
 	Metrics() *monitoring.Registry       // MetricSet specific metrics
+	Logger() *logp.Logger                // MetricSet specific logger
```

Comment:
I like the idea, though I would have went with a shorter `Log()` name in effort to keep the code lines shorter like (`ms.Log().Info(...)`).

Reply:
@andrewkroh I like that, will open a follow up PR.

Reply:
Name updated here: https://github.com/elastic/beats/pull/11126